app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -104,9 +104,14 @@ export class MergeConflictsDialog extends React.Component<
    *  commits the merge displays the repository changes tab and dismisses the modal
    */
   private onSubmit = async () => {
+    // filter out untracked files so we don't commit them
+    const trackedFiles = this.props.workingDirectory.files.filter(f => {
+      return f.status.kind !== AppFileStatusKind.New
```

Comment:
Is there a way we can capture this as a regression test? I'm worried that this also catches `'added'` entries from here (which might be valid as part of the merge commit):

https://github.com/desktop/desktop/blob/b556ff8f9746447b7457fdbe78796b835e1db322/app/src/lib/git/status.ts#L104-L113

Reply:
ah good catch! i will adjust to cover that case and add some tests too